sdk/src/main/scala/com/horizen/SidechainState.scala

```diff
@@ -288,9 +285,11 @@ class SidechainState private[horizen] (stateStorage: SidechainStateStorage,
       if (tx.isInstanceOf[OpenStakeTransaction]) {
         if (!params.restrictForgers)
```

Comment:
Do we still need this check? Since it verified just below inside the `if (isForgingOpen())` ?
The new exception message satisfies both cases as for me.

Reply:
Ok i remove it.